### PR TITLE
CNV-23499: 4.10+ HPP CSI only - content

### PR DESCRIPTION
**Pull request by aspauldi**
Version(s): 4.11, 4.12, 4.13

Issue: [CNV-23499: 4.10+ HPP CSI only - content](https://issues.redhat.com/browse/CNV-23499)

Links to docs previews:

-[Creating a storage class for the CSI driver with the storagePools stanza](https://57422--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.html#virt-creating-storage-class-csi-driver_virt-configuring-local-storage-for-vms)--removed prerequisite requiring version 4.10 and references to legacy provisioner

-[Configuring local storage for virtual machines](https://57422--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.html)--moved content from **About the hostpath provisioner** module into this assembly, removed the **About the hostpath provisioner** module, and removed paragraphs related to 4.10

-[Installing OpenShift Virtualization using the CLI](https://57422--docspreview.netlify.app/openshift-enterprise/latest/virt/install/installing-virt-cli.html)-- corrected broken xref in **Next steps**

-[Installing OpenShift Virtualization using the web console](https://57422--docspreview.netlify.app/openshift-enterprise/latest/virt/install/installing-virt-web.html)----corrected broken xref in **Next steps**

QE review:
- [x] QE has approved this change. (LGTM from Kevin in corresponding Jira.)

Additional information:

There are two additional PRs related to this one: 

[#57514](https://github.com/openshift/openshift-docs/pull/57514)--to repeat in the 4.13 release notes the removal of support for the legacy HPP in version 4.12

[https://github.com/openshift/openshift-docs/pull/57355](https://github.com/openshift/openshift-docs/pull/57355)--to move discontinuation of legacy HPP support from Deprecated to Removed feature in 4.12 release notes

### CODE REVIEW

File: virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc [hunk 1]
```diff
@@ -1,23 +1,21 @@
 :_content-type: ASSEMBLY
-[id="virt-configuring-local-storage-for-vms"]
+[id=""]
```
Comment (sjhala-ccs) [inline virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc:2]: The assembly needs to have a unique ID per our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata)

File: modules/virt-about-hostpath-provisioner.adoc [hunk 2]
```diff
@@ -1,28 +1,11 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
```
Comment (sjhala-ccs) [inline modules/virt-about-hostpath-provisioner.adoc:3]: Module metadata [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)

**Review by sjhala-ccs — COMMENTED** [review]
A couple of things I noticed that could be causing the build to fail

File: virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc (hunk 1 above)
Comment (aspauldi) [reply virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc:2]: Thanks, @sjhala-ccs. How do I find out what the missing id should be?

File: modules/virt-about-hostpath-provisioner.adoc (hunk 2 above)
Comment (aspauldi) [reply modules/virt-about-hostpath-provisioner.adoc:3]: I looked at the guidelines, but I'm not sure about how to proceed. I thought that I had found all xref instances and fixed them, but now I'm seeing this for each of the virt folders:

image

File: virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc (hunk 1 above)
Comment (sjhala-ccs) [reply virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc:2]: The assembly ID is usually the file name. So, in this case:
````
[id="virt-configuring-local-storage-for-vms"]
````
That is what it was before as you can see by the red highlighting above.

File: modules/virt-about-hostpath-provisioner.adoc (hunk 2 above)
Comment (aspauldi) [reply modules/virt-about-hostpath-provisioner.adoc:3]: Done! Thanks

**Comment by aspauldi** [comment]
Hello @aglitke,  can you please review this PR? Thanks.

**Comment by bmcelvee** [comment]
LGTM!

**Review by kgoldbla — COMMENTED** [review]
lgtm

**Comment by jboxman-rh** [comment]
@aspauldi, does this also go into 4.10 or not? I can't tell for sure. Thanks!

**Pull request merged**